aiida_wannier90/io/__init__.py

```diff
@@ -7,4 +7,4 @@
 This submodule contains helper functions to create input files.
 """
 
-from ._write_win import write_win
+#from ._write_win import write_win
```

Comment:
What's the reason for this change? Note that this changes the public API: previously, the helper function was publicly accessible as ``aiida_wannier90.io.write_win``. Now it would be an internal - only function.